Sources/iPad/View Controllers/File/DocumentsViewController_iPad.m

```diff
@@ -123,14 +123,14 @@ - (void)showImagePickerForAddPhotoAction:(UIImagePickerController *)picker {
         rect.size.width = 0;
         rect.size.height = 0;
         // workaround will be fixed by https://jira.exoplatform.org/browse/MOB-1828
-        [[NSOperationQueue mainQueue] addOperationWithBlock:^{
+        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
No reason, I just usually use the dispatch for this situation (get main queue), I've never used so I just make a try to see if there are different.